stdlib/typing.pyi

```diff
@@ -129,8 +129,8 @@ if sys.version_info >= (3, 11):
 if sys.version_info >= (3, 12):
     __all__ += ["TypeAliasType", "override"]
 
-ContextManager = AbstractContextManager
-AsyncContextManager = AbstractAsyncContextManager
+ContextManager = AbstractContextManager[_T_co, bool | None]  # noqa: Y026
+AsyncContextManager = AbstractAsyncContextManager[_T_co, bool | None]  # noqa: Y026
```

Comment:
I assume that this change is the reason that mypy primer stumbles when checking kopf:

https://github.com/nolar/kopf/blob/96e8cbcec16c78be177922f5d8c000fa4b4022c1/kopf/_core/engines/admission.py#L266

```python
    elif isinstance(settings.admission.server, AsyncContextManager):
```

Just adding generic parameters here seems wrong to me as this is a simple name aliasing, not a type alias. Why are those changes necessary?

Reply:
Refer to my previous comment: https://github.com/python/typeshed/pull/11048#issuecomment-2012965236

This is the third iteration of this change, both using a subclass or completely copying the protocol with just the one generic parameter seem to make stubtest really unhappy for some reason. It starts complaining that `ContextManager`/`AsyncContextManager` aren't types.

Leaving the old aliases intact is a possibility, however, we then end up with a false negative if people use the second parameter on `ContextManager`/`AsyncContextManager`, since that will result in a runtime error, due to restrictive implementation of `typing.Generic`, which checks the number of parameters passed in.

Reply:
I suppose the stubtest errors can actually be ignored. I think the reason why stubtest didn't complain before, is, because it was just a regular assignment. Technically the complaint is accurate `typing._SpecialGenericAlias` is not a type, I just assumed stubtest already special cases these aliases, but apparently it doesn't, because it wasn't necessary for the simple re-exports.

So I will revert to the previous solution, which was more sane.